db/seeds/search_module.rb

```diff
@@ -18,7 +18,6 @@
 SearchModule.create(tag: 'OSPEL', display_name: 'Spelling Suggestions (Search.gov Images)')
 SearchModule.create(tag: 'AWEB', display_name: 'Web Results Only (Azure)')
 SearchModule.create(tag: 'FRDOC', display_name: 'Federal Register Documents')
-SearchModule.create(tag: 'DECOR', display_name: 'Value-added Links for SEC Filings')
```

Comment:
Just a comment that this should bump https://cm-jira.usa.gov/browse/SRCH-2695 up in the queue. :-)